.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -61,12 +61,10 @@ body:
       label: 📟 Stack Trace
       description: Copy and paste the stack trace from FluentHub's error page.
       value: |
-        "details><summary>Stack trace details</summary><div>
-        
```

Comment:
Why did you delete double line breaks and double-quatations?

Reply:
why shouldn't I have deleted quotation mark?

Reply:
Ah, Already '|' is there. Never mind.

> How about 'Why did you delete double line breaks?'

Reply:
> How about 'Why did you delete double line breaks?'

Why was it necessary in the first place?

Reply:
It doesnt seem to be recongnized as a code block.

Reply:
Here

![image](https://user-images.githubusercontent.com/62196528/193080472-f5d38869-9da2-4162-9b0e-ca219893daa5.png)


![image](https://user-images.githubusercontent.com/62196528/193080425-3496d5ac-6457-41e2-81a2-107ae9886b6c.png)